src/components/HeaderBar/HeaderBar.module.scss

```
@@ -38,18 +42,27 @@
   letter-spacing: 0.04em;
   font-family: v(ff-sans);
   line-height: 150%;
-  font-weight: 600;
+  font-weight: 500;
 
   justify-content: flex-start;
 
-  @include media('<tablet') {
+  @include media('<desktop') {
     font-size: rem(12px);
   }
 
   @include media('<phone') {
     font-size: rem(10px);
   }
 }
+.text {
+  @include media('<desktop') {
+    &:after {
+      content: '>';
```

Very small nitpick, we should probably have a space before the arrow.

```suggestion
      content: ' >';
```